src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Shared/DraftApprenticeship.cshtml

```diff
@@ -168,7 +168,7 @@ else
 <div class="govuk-form-group  @Html.AddClassIfPropertyInError(x => x.StartDate, "govuk-form-group--error")">
     <fieldset class="govuk-fieldset" aria-describedby="start-date-hint" role="group">
         <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
-            Planned training start date
+            Planned apprenticeship training start date
```

Comment:
Are these new field names just for the flexible apprenticeships. Ie do the old fieldnames renames for regular apprenticeships?